config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  root to: "gatherings#index"
+  root to: redirect("gatherings")
```

Comment:
Can we actually delete the `home_controller.rb` file, or is it still necessary for some behind-the-scenes reason?

Reply:
We can kill it